mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -146,6 +151,55 @@ class Word2VecModel private[ml] (
     wordVectors: feature.Word2VecModel)
   extends Model[Word2VecModel] with Word2VecBase {
 
+
+  /**
+   * Return a map of every word to its Vector representation.
+   */
+  val getVectors: Map[String, Array[Float]] = wordVectors.getVectors
```

Comment:
So its my understanding, from my reading of the code, it isn't possible to access the Spark context here. If we look at similar methods in other models (like predict in classifier), they can't access the Spark Context, its only when we are applying fit/transform and we have a dataframe passed into us that we can access the SparkContext.